code/cutscene/ogg/OggDecoder.cpp

```diff
@@ -233,7 +234,7 @@ namespace cutscene
                 vorbis_block_init(&movie.vstate, &movie.vblock);
             }
 
-            auto fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
+            fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
```

Comment:
I adapted the old Ogg loader and didn't bother cleaning it up. I'll remove the goto stuff which should fix the declaration issues.